skgarden/quantile/ensemble.py

```diff
@@ -129,16 +131,29 @@ def predict(self, X, quantile=None):
         if quantile is None:
             return super(BaseForestQuantileRegressor, self).predict(X)
 
-        sorter = np.argsort(self.y_train_)
+        if self.n_outputs_ > 1:
+            sorter = np.argsort(self.y_train_, axis=0)
+        else:
+            sorter = np.argsort(self.y_train_)
```

Comment:
Hi @knefati, I'm happy to see you fixed the multi-output! I used the library in 2018 and needed this for my multi step prediction problem. I wonder if the sorter variable has to be defined here at all since the weighted_percentile function performs the sorting if no sorter is given anyway (this applies to the original solution as well).

Would be very thankful for a response!